src/main/kotlin/uk/gov/dwp/dataworks/services/CredentialsService.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.dwp.dataworks.services
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.auth.credentials.AwsSessionCredentials
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider
+
+@Service
+class CredentialsService{
```

Comment:
this was created so we can switch to testing it locally using the session credentials

Reply:
can that not be achieved in the setup and mocking within the tests?

Reply:
when we said testing it locally we meant running the application locally so we want to keep this class to easily switch when we run in our local machine

Reply:
We likely don't even have to mock - injecting env vars in the tests which requires them would also achieve this.
I would also question any unit test which hits the real AWS API instead of mocking it's behaviour. Ideally any touch points outside the app would be in `integration` tests instead, which we don't have in this case. we're likely to create needy tests or introduce instability  if we hit a real api from a local / CI test environment.